src/background.ts

```diff
@@ -1131,8 +1138,11 @@ app.on("ready", async () => {
   ) {
     log.info("VOICEVOX already running. Cancelling launch.");
     log.info(`File path sent: ${filePath}`);
+    appState.willQuit = true;
     app.quit();
     return;
+  } else if (filePath?.endsWith(".vvpp")) {
+    await installVvppEngine(filePath);
   }
```

Comment:
別にこれでも挙動は正しいのですが、ここをelseにすると、「多重起動じゃなかった場合の特別な処理」という意図と受け取られるかもです。
多重起動と独立した処理なので、ブロックは分けたほうがわかりやすいかも？
```suggestion

  if (filePath?.endsWith(".vvpp")) {
    await installVvppEngine(filePath);
  }
```

（別にそのままでも問題ないと思います！）

Reply:
確かに、returnされるのでelseなくても大丈夫ですね。修正しました